ecs/guardian.conf

```diff
@@ -1,6 +1,3 @@
-[SERVICE]
```

Comment:
Believe we no longer needs to specify a Service. the init process will recognize guardian-parsers.conf as a parsers file and create a service for us.